Extensions/TriggerOnceConditionForObject.json

```
@@ -0,0 +1,338 @@
+{
+  "author": "",
+  "description": "## Summary\nThis extension is useful if you want to trigger once per instance.\nThis is simpler than creating your own instance variables.\n\n## How to use: \n- Adding the **Trigger Once Condition** behavior to an object.\n- In the events add the **Trigger once per instance** condition to the instance.\n\nThe condition will be identified by the **condition name** and will be executed only once until you reset if you want using it one more time.\n\nThis extension add:\n\n- **Trigger Once Condition** - behavior\n- **Trigger once per instance** - condition\n- **Reset all conditions** - action\n- **Reset condition** - action",
+  "extensionNamespace": "",
+  "fullName": "Trigger Once Condition for Object",
+  "helpPath": "https://github.com/PANDAKO-GitHub/TriggerOnceConditionForObject",
+  "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLXJlcGVhdC1vbmNlIiB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCI+PHBhdGggZD0iTTEzLDE1VjlIMTJMMTAsMTBWMTFIMTEuNVYxNU0xNywxN0g3VjE0TDMsMThMNywyMlYxOUgxOVYxM0gxN003LDdIMTdWMTBMMjEsNkwxNywyVjVINVYxMUg3VjdaIiAvPjwvc3ZnPg==",
+  "name": "TriggerOnceConditionForObject",
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/repeat-once.svg",
+  "shortDescription": "This behavior adds a \"Trigger once per instance\" condition to the object (instance).",
+  "version": "1.0.3",
+  "tags": [
+    "once",
+    "object",
+    "variable",
+    "instance"
+  ],
+  "authorIds": [
+    "Zu55H5hcb9YmZTltIVOTAFDJQyB2"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [],
+  "eventsBasedBehaviors": [
+    {
+      "description": "Adds a condition to the object that will be executed trigger once per instance.",
+      "fullName": "Trigger Once Condition",
+      "name": "TriggerOnceCondition",
+      "objectType": "",
+      "eventsFunctions": [
+        {
+          "description": "The condition name of this instance was checked for the first time.\nUnlike the \"Trigger once while true\" condition, this condition stays false until reset.",

```

```suggestion
          "description": "The condition name of this instance was checked for the first time.\nUnlike the \"Trigger once while true\" condition, this condition stays false until reset manually with the appropriate action.",

```

It's automatic or not, an innocent user doesn't know it, it must be specified.